src/App.test.js

```diff
@@ -1,8 +1,44 @@
+import React from 'react';
 import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import { MemoryRouter } from 'react-router';
 import App from './App';
 
-test('renders learn react link', () => {
-  render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
+describe('App render all and test', () => {
+  beforeEach(() => {
+    render(
+      <MemoryRouter>
+        <App />
+      </MemoryRouter>,
+    );
+  });
+
+  test('if NavBar links exist', () => {
+    expect(screen.getByLabelText('Calculator')).toBeInTheDocument();
+    expect(screen.getByLabelText('Quote')).toBeInTheDocument();
+    expect(screen.getByLabelText('Home')).toBeInTheDocument();
+    screen.debug();
+  });
+
+  test('Quote link and check the owner of the quote', () => {
+    userEvent.click(screen.getByLabelText('Quote').firstChild);
+    screen.debug();
+    expect(
+      screen.getByText('-Raymond L. Wilder, 1968'),
+    ).toBeInTheDocument();
+  });
+
+  test('to open Calculate page and click all calculator buttons', () => {
+    userEvent.click(screen.getByLabelText('Calculator').firstChild);
+    screen.debug();
+
+    const cells = document.querySelectorAll('.cell');
+    console.log(cells[0].innerHTML);
+    cells.forEach((cell) => userEvent.click(cell));
+    const resultElement = document.querySelector('#calculate-result');
+    console.log(resultElement.innerHTML);
+
+    expect(resultElement.innerHTML).toBe('0');
+  });
```

Comment:
- [ ] It would be great to see a cleaner test.
- [ ] You could start removing the console.log

![Screen Shot 2022-03-23 at 17 35 39](https://user-images.githubusercontent.com/67211919/159813813-84cd129c-6141-4634-95d0-7f07da1e7c68.png)